lib/dependencies/ContextDependencyHelpers.js

```diff
@@ -84,7 +86,17 @@ ContextDependencyHelpers.create = (
 		);
 		dep.loc = expr.loc;
 		const replaces = [];
-		if (prefixRange && prefix !== prefixRaw) {
+		const templateLiteral =
```

Comment:
You can walk all `parts` of the `param`.

* If it's a constant string: replace the `range` with the string (push to `replaces`)
* If it's not constant: walk it with the parser